WebRoot/js/ajax/dwt/xforms/XFormItem.js

```diff
@@ -994,14 +994,14 @@ XFormItem.prototype.handleKeyPressDelay = function(ev, domItem) {
 };
 
 XFormItem.prototype.getKeyPressHandlerHTML = function () {
-	// if AjxEnv.isNav, etc.
-	var keydownEv = DwtEvent.ONKEYPRESS;
+  // if AjxEnv.isNav, etc.
+  var keydownEv = DwtEvent.ONKEYPRESS; // "onkeypress"
```

Comment:
No I'm not fixing func/log issue.  Some how some one made the changes before I made my pull request.  So there's no code changes other than the comments.  But I'd still like to keep the comments though.